sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/model/ThrottlingRetryOptions.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos;
+package com.azure.cosmos.model;
```

Comment:
I am not sure if this is a model. as it specifies client behaviour not over wire contract

Reply:
Agreed, as it specifies client behavior, will move it back.